refactor: fix cli seeder generate and add seeder env vars

Langstra opened the pull request:
Closes #2310

B4nan reviewed (changes requested):
looking good, please also update the list of supported env vars in https://mikro-orm.io/docs/configuration/#using-environment-variables

Langstra commented:
> looking good, please also update the list of supported env vars in https://mikro-orm.io/docs/configuration/#using-environment-variables
Added those [here](https://github.com/mikro-orm/mikro-orm/pull/2370/files#diff-26a27f31b938efc2e2e661c3ee8a5b46ea02efaf7a89e9d38d892878da24e29d), right? 
> 
> btw i consider those as just chores/refactors, as such commits should not be part of the v5.0 changelog (seeder is a new feature, it does not make sense to have fixes for things that were not yet released as stable)
Updated the commit message.
